compiler/resolution/preFold.cpp

```diff
@@ -231,6 +231,40 @@ static Expr* preFoldPrimOp(CallExpr* call) {
     break;
   }
 
+  case PRIM_POPULATE_TESTS: {
+    int  totalTest = 0;
+    forv_Vec(FnSymbol, fn, gFnSymbols) {
+      if (fn->throwsError()) {
+        const char* name = astr(fn->name);
```

Comment:
Since `name` and `tryCall` are only really useful if we know we're going to call `tryResolveCall`, in the interest of limiting symbol scope and avoiding unnecessary computations I would move these lines and the `tryCall->remove()` inside the if statement on what is currently line 252.

Reply:
Updated it.